manage-data/lifecycle/index-lifecycle-management/rollover.md

```diff
@@ -9,50 +9,96 @@ products:
+## Recommended approaches
 
-::::{tip}
-Rolling over to a new index based on size, document count, or age is preferable to time-based rollovers. Rolling over at an arbitrary time often results in many small indices, which can have a negative impact on performance and resource usage.
-::::
+Decide your approach to index rotation based on your use case and requirements.
 
+| Use case               | Recommended approach                                      | Setup benefits and limitations                                                                  |
+| ---------------------- | --------------------------------------------------------- | ----------------------------------------------------------------------------------------------- |
+| Logs, metrics          | [Data streams](rollover.md#rollover-data-stream)          | Configure rollover with lifecycle management, *minimal setup*, control over rollover timing ^1^ |
+| Legacy indexing setup  | [Alias-based rollover](rollover.md#rollover-with-aliases) | Configure rollover with lifecycle management, *advanced setup*, control over rollover timing    |
+| Small, static datasets | No rollover                                               | Simpler management                                                                              |
 
-::::{important}
-Empty indices will not be rolled over, even if they have an associated `max_age` that would otherwise result in a roll over occurring. A policy can override this behavior, and explicitly opt in to rolling over empty indices, by adding a `"min_docs": 0` condition. This can also be disabled on a cluster-wide basis by setting `indices.lifecycle.rollover.only_if_has_documents` to `false`.
-::::
+^1^ Rollover is handled automatically in {{es-serverless}} projects. {applies_to}`serverless: ga`
 
+:::{tip}
+For new projects, use data streams. They're simple to manage with lifecycle policies where you define phases and actions that handle rollover automatically.
+:::
 
-::::{important}
-The rollover action implicitly always rolls over a data stream or alias if one or more shards contain 200000000 or more documents. Normally a shard will reach 50GB long before it reaches 200M documents, but this isn’t the case for space efficient data sets. Search performance will very likely suffer if a shard contains more than 200M documents. This is the reason of the builtin limit.
-::::
 
+### Rotating your indices with data streams [rollover-data-stream]
+
+We recommend using [data streams](../../data-store/data-streams.md) to manage time series data. When set up to use an ILM policy that includes rollover, a data stream automatically manages the rotation of your indices. This ensures you can write to the data stream without additional configuration.
```

Comment:
I think this could be made more concise:

```suggestion
We recommend using [data streams](../../data-store/data-streams.md) to manage time series data. When set up to use an {{ilm-init}} policy that includes rollover, a data stream manages the rotation of your indices without additional configuration.
```